src/BUILD

```diff
@@ -145,7 +144,7 @@ rule_size_test(
     # WARNING: Only adjust the number in `expect` if you are intentionally
     # adding or removing embedded tools. Know that the more embedded tools there
     # are in Bazel, the bigger the binary becomes and the slower Bazel starts.
-    expect = 431,
+    expect = 445,
```

Comment:
The diff is actually 465 to 445.  The 431 you may be seeing on the left is from an intermediate step where I brought it down too much.

Reply:
431 is actually the number at HEAD. you should probably adjust this number further

Reply:
Yes. I see that now Alex's removal of the android tools happened after my last sync from head, so I was based off the old value.